assets/css/about.css

```diff
@@ -0,0 +1,478 @@
+  background-size: cover;
+}
+
+#summit2 {
+  background: url('../img/summit2.jpeg') no-repeat;
+  background-size: cover;
+}
+
+@media screen and (max-width: 768px) {
+  #summits {
+    padding: 50px 20px;
+  }
+
+  .summits-heading h2 {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 20px;
+  }
+
+  .summits-heading h2::after {
+    margin-left: 120px;
+  }
+
+  .summits {
+    flex-direction: column;
+  }
+
+  .summit {
+    width: 100%;
+  }
+
+  #summit1 {
+    margin-bottom: 24px;
+  }
+
+  #summit2 {
+    margin-bottom: 0;
+  }
+}
+
+#partners-about {
+  /* background: #272a31; */
+  background: #000;
+  display: none;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 40px 140px;
+  margin-top: 50px;
+}
+
+.partners-header h2 {
+  color: #fff;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 30px;
+  letter-spacing: 0;
+}
+
+.partners-header h2::after {
+  content: '';
+  display: block;
+  position: absolute;
+  width: 50px;
+  border-bottom: 3px solid #ec5242;
+  margin-left: 22px;
+  margin-top: 10px;
+}
+
+.partners {
+  margin-top: 50px;
+  margin-bottom: 30px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.partners img {
+  height: 35px;
+  width: auto;
+  margin-right: 35px;
+}
+
+@media screen and (max-width: 768px) {
+  #partners-about {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: center;
+    padding: 20px 20px;
+    margin-top: 50px;
+  }
+
+  .partners-header h2 {
+    color: #fff;
+    font-family: 'Cocogoose', sans-serif;
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 30px;
+    letter-spacing: 0;
+  }
+
+  .partners-header h2::after {
+    content: '';
+    display: block;
+    position: absolute;
+    width: 50px;
+    border-bottom: 3px solid #ec5242;
+    margin-left: 22px;
+    margin-top: 10px;
+  }
+
+  .partners {
+    margin-top: 50px;
+    margin-bottom: 30px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .partners img {
+    height: 35px;
+    width: auto;
+    margin-right: 35px;
+    margin-bottom: 40px;
+  }
+}
+
+.footer-about {
+  background: #000;
+  padding: 30px 100px 30px 300px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer-about .footer-logo {
+  margin-right: 40px;
+}
+
+.footer-about .footer-logo img {
+  height: 40px;
+}
+
+#blacklogo {
+  display: none;
+}
+
+.footer-about .footer-info h5 {
+  color: #fff;
+  font-family: 'Cocogoose', sans-serif;
+  font-weight: 500;
+  font-size: 18px;
+  margin-bottom: 12px;
+}
+
+.footer-about .footer-info p {
+  color: #fff;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
- [ ] Kindly, please redesign your app using a mobile-first approach, as learned in the whole module that we finish now.